testing/mock/test_system.cpp

```diff
@@ -396,6 +401,9 @@ uint32_t get_device_id(const std::string &sysclass) {
 }
 
 int test_system::open(const std::string &path, int flags) {
+  if (!initialized_) {
```

Comment:
Ah.. good catch.

Reply:
Discovered this testing the embedded Python 🐍